sdk/identity/identity/src/tokenCache/TokenCachePersistence.ts

```diff
@@ -31,11 +32,20 @@ export class TokenCachePersistence {
       retryDelay: 50
     };
 
+    let extensions: any;
+    try {
+      /* eslint-disable-next-line @typescript-eslint/no-require-imports */
+      extensions = require("@azure/msal-node-extensions");
```

Comment:
Does this cause any issues with bundling if `@azure/msal-node-extensions` isn't available?

Reply:
Let me check

Reply:
I couldn't find any issue bundling!